WordPress/Classes/Services/PostRepository.swift

```diff
@@ -217,6 +217,12 @@ final class PostRepository {
         let original = post.original()
         var changes = changes
 
+        // Scheduled posts need to sync with a status of 'publish'. If you omit
+        // the status, it will move the post to `draft`.
+        if original.status == .scheduled && service is PostServiceRemoteXMLRPC {
```

Comment:
I hope to remove it after switching to `wp.editPost`, assuming it works correctly and doesn't automatically transition the post to `draft`.

Reply:
I reverted this fix, as it's clear we are going with `wp.editPage`, so it's no longer needed.